apps/prairielearn/src/lib/db-types.ts

```diff
@@ -346,6 +346,7 @@ export const CourseInstanceSchema = z.object({
   id: IdSchema,
   json_comment: JsonCommentSchema.nullable(),
   long_name: z.string().nullable(),
+  share_source_publicly: z.boolean().nullable(),
```

Comment:
This is not nullable; check your migration.

Reply:
thanks, done in dd2e51b894